index.html

```diff
@@ -12,8 +12,164 @@
 
 <body>
 
-<h1>test Font</h1>
-<p>Lorem ipsum dolor sit amet.</p>
+<header>
+    <!--Nav staat los van het outer en inner container principe. Nav houdt wel de breedte -->
+    <nav class="outer-container">
+        <ul class="nav-links-container">
+            <li><a href="#portfolio">Case Studies</a></li>
+            <li><a href="#services">Services</a></li>
+            <li><a href="#contact">Contact Us</a></li>
+        </ul>
+    </nav>
+    <!--    Outer en inner container principe begint bij dit deel van de header-->
+    <div class="outer-container">
+        <div class="inner-container-sm header-cta">
+            <h1>Your website is an investment in the success of your business.
+            </h1>
+            <p class="header-subtitle">We are here to help you succeed.</p>
+            <button type="button">Lets' work together</button>
+            <img class="squiggle-line-stripe" src="assets/squiggle_line_stripe.svg"
+                 alt="squiggle-line-stripe">
+        </div>
+    </div>
+</header>
+<div class="header-triangle"></div>
 
+<section class="hello outer-container">
+    <div class="inner-container-sm">
+        <div class="hello-content-container">
+            <h2>Rekupe is a digital agency in Los Angeles.
```

Comment:
Headings worden gebruikt om een bepaalde hiërarchie in de titels weer te geven. 

De belangrijkste titel van je pagina, daar kan er maar één van zijn, krijgt een h1-element. Daarna krijgen alle verschillende secties een h2-element. Zo ga je steeds verder naar beneden, en als het goed is, zou je hier dan op een ander element uitkomen dan een h2-element, want de verschillende secties zouden dan al deze in gebruik nemen. 

Ik vergelijk het ook altijd met een boek. Een boek heeft één titel, en elk hoofdstuk zou dan een h2-element zijn. Heb je nog paragrafen met koppen, dan zou dat een h3 zijn, enzovoorts.